scss/components/side-nav.scss

```diff
@@ -100,6 +100,8 @@ $block: #{$fd-namespace}-side-nav;
     }
 
     &__link {
+        display: flex;
+        align-items: center;
```

Comment:
should this go higher? 

`&__link,&__sublink`, line 75

Reply:
I tried to put higher, please see screenshot:
<img width="283" alt="Screen Shot 2019-06-14 at 9 17 36 AM" src="https://user-images.githubusercontent.com/41170611/59512055-deb96f80-8e85-11e9-8245-2601d05ccb2d.png">

and it showed result as:
![Screen Shot 2019-06-14 at 9 17 43 AM](https://user-images.githubusercontent.com/41170611/59512181-32c45400-8e86-11e9-8cb5-e2ab4f6f6ea4.png)

so i think we should keep as before:
<img width="244" alt="Screen Shot 2019-06-14 at 9 26 32 AM" src="https://user-images.githubusercontent.com/41170611/59512352-89ca2900-8e86-11e9-8abb-c60bb01dc767.png">

Reply:
did you leave the `display: block;` on line 79?

Reply:
yes, i did not change it.

Reply:
um, you would have to remove that. thats how the cascade works.

Reply:
ok, i fixed it, and now the code is at higher position and After screenshot:
![Screen Shot 2019-06-14 at 3 52 54 PM](https://user-images.githubusercontent.com/41170611/59534506-80f44a00-8ebc-11e9-9480-594730097eaf.png)

please let me know if that is ok now? thanks.